File: backend/src/main/java/com/team05/petmeeting/domain/naming/repository/NameVoteHistoryRepository.java
package com.team05.petmeeting.domain.naming.repository;

import com.team05.petmeeting.domain.naming.entity.NameVoteHistory;
import org.springframework.data.jpa.repository.JpaRepository;

public interface NameVoteHistoryRepository extends JpaRepository<NameVoteHistory, Long> {
    Boolean existsByUserIdAndAnimalId(Long userId, Long animalId);
}

File: backend/src/main/java/com/team05/petmeeting/domain/naming/repository/NameVoteHistoryRepository.kt
package com.team05.petmeeting.domain.naming.repository

import com.team05.petmeeting.domain.naming.entity.NameVoteHistory
import org.springframework.data.jpa.repository.JpaRepository

interface NameVoteHistoryRepository : JpaRepository<NameVoteHistory, Long> {
    fun existsByUserIdAndAnimalId(userId: Long, animalId: Long): Boolean
}
